mnefun/_mnefun.py

```diff
@@ -801,7 +844,7 @@ def run_sss_command(fname_in, options, fname_out, host='kasga', port=22,
             pass
 
 
-def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
+def run_sss_positions(fname_in, fname_out, host, port=22, opts=''):
```

Comment:
If you have your `routes` configured properly to work on UW's network it should. Please revert